### DISCUSSION TITLE
Linux Mint install error

### DISCUSSION THREAD

**Discussion by appmakerusa** [quackduck/uniclip#9] [General]
Install to my Mac worked but install to Linux failed.

==> Tapping quackduck/tap
Cloning into '/home/me/.linuxbrew/Homebrew/Library/Taps/quackduck/homebrew-tap'...
remote: Enumerating objects: 138, done.
remote: Counting objects: 100% (138/138), done.
remote: Compressing objects: 100% (132/132), done.
remote: Total 138 (delta 68), reused 2 (delta 0), pack-reused 0
Receiving objects: 100% (138/138), 23.22 KiB | 642.00 KiB/s, done.
Resolving deltas: 100% (68/68), done.
**Error: Invalid formula: /home/me/.linuxbrew/Homebrew/Library/Taps/quackduck/homebrew-tap/cool.rb
formulae require at least a URL
Error: Cannot tap quackduck/tap: invalid syntax in tap!**

**Comment by quackduck** [comment]
Hello! This happens because one of the formulae in my tap (`cool`) is not available for linux.

Try using this command: `brew install quackduck/tap/uniclip` or manually install it using the path to the formula: `brew install /path/to/formula` (most probably `brew install /home/me/.linuxbrew/Homebrew/Library/Taps/quackduck/homebrew-tap/uniclip.rb`)

Alternatively you can build from source :)

**Comment by quackduck** [comment]
Sorry for replying so late, I didn't notice this notification cause it was covered in other ones lol
